### PR TITLE
[builder] Perform file IO with templefiles

### PR DESCRIPTION
**Pull request by jaylees14**
```
sbt:temple> run generate -o my-project src/test/scala/temple/testfiles/simple.temple
```

```
~/u/y/t/temple ❯❯❯ tree my-project
my-project
└── User-db
    └── init.sql
```
```
~/u/y/t/temple ❯❯❯ cat my-project/User-db/init.sql
CREATE TABLE User (
  username TEXT,
  email VARCHAR(40) CHECK (length(email) >= 5),
  firstName TEXT,
  lastName TEXT,
  createdAt TIMESTAMPTZ,
  numberOfDogs INT,
  yeets BOOLEAN UNIQUE,
  currentBankBalance REAL CHECK (currentBankBalance >= 0.0),
  birthDate DATE,
  breakfastTime TIME
);
CREATE TABLE Fred (
  field TEXT
);              
```

Will add end to end testing in another PR since that will be a lot of infra changes

### CODE REVIEW

File: src/main/scala/temple/builder/project/ProjectBuilder.scala [hunk 1]
```diff
@@ -19,12 +21,12 @@ object ProjectBuilder {
     val databaseCreationScripts = templefile.services.map {
       case (name, service) =>
         val queries: Seq[Statement.Create] = DatabaseBuilder.createServiceTables(name, service)
-        val generatedScripts: Seq[String] = service.lookupMetadata[Database].getOrElse(Postgres) match {
+        service.lookupMetadata[Database].getOrElse(Postgres) match {
           case Postgres =>
             implicit val context: PostgresContext = PostgresContext(QuestionMarks)
-            queries.map(PostgresGenerator.generate)
+            val postgresStatements                = queries.map(PostgresGenerator.generate).mkString("\n")
```
Comment (c-harding) [inline src/main/scala/temple/builder/project/ProjectBuilder.scala]: Can we rename `queries` to `createStatements`?

**Review by c-harding — DISMISSED** [review]

**Review by c-harding — DISMISSED** [review]
plz no dismiss

**Review by willjvsmith — APPROVED** [review]
LGTM 🚀

**Review by c-harding — APPROVED** [review]
LG2M

**Pull request merged**